### PR TITLE
fix: responsive image challenge

### LINKED ISSUE #37875: The "Make an Image Responsive" challenge is missing a CSS attribute
**Describe your problem and how to reproduce it:**
In the *Responsive Web Design Principles: Make an Image Responsive* challenge, the CSS attribute `width: 100%` is missing from the code and description, which it may could not affects the challenge result, but it won't resize the image to 100% size as described into the description.

**Add a Link to the page with the problem:**
https://github.com/freeCodeCamp/freeCodeCamp/blob/master/curriculum/challenges/english/01-responsive-web-design/responsive-web-design-principles/make-an-image-responsive.english.md

**Tell us about your browser and operating system:**
* Browser Name: Brave
* Browser Version: 1.0.1 Chromium: 78.0.3904.108 (Official build) (64-bit)
* Operating System: Windows 10 x64

**If possible, add a screenshot here (you can drag and drop, png, jpg, gif, etc. in this box):**
brave_2019-12-05_18-52-17
brave_2019-12-05_18-52-39

### PR DESCRIPTION
**Pull request by moT01**
We didn't exactly agree on a solution to the issue, I decided to just make a PR and see if we can tweak it from here if needed.

- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] All the files I changed are in the same world language (for example: only English changes, or only Chinese changes, etc.)
- [x] My changes do not use shortened URLs or affiliate links.

Closes #37875

The [forum hint solution](https://www.freecodecamp.org/forum/t/freecodecamp-challenge-guide-make-an-image-responsive/301140) will need to be changed if this is merged.

### CODE REVIEW

File: curriculum/challenges/english/01-responsive-web-design/responsive-web-design-principles/make-an-image-responsive.english.md [hunk 1]
```diff
@@ -68,13 +72,17 @@ tests:
...
-   display: block;
-   height: auto;
- }
+.responsive-img {
+  max-width: 100%;
+  height: auto;
+}
+
+img {
+  width: 500px;
```
Comment (ojeytonwilliams) [inline curriculum/challenges/english/01-responsive-web-design/responsive-web-design-principles/make-an-image-responsive.english.md]:
```suggestion
  width: 600px;
```
Unfortunately we can't be sure it overflows, but maybe make it a little bigger to make it more obvious.

File: curriculum/challenges/english/01-responsive-web-design/responsive-web-design-principles/make-an-image-responsive.english.md [hunk 2]
```diff
@@ -51,9 +47,17 @@ tests:
 
 ```html
 <style>
+.responsive-img {
+
+
+}
 
+img {
+  width: 500px;
```
Comment (ojeytonwilliams) [inline curriculum/challenges/english/01-responsive-web-design/responsive-web-design-principles/make-an-image-responsive.english.md]:
```suggestion
  width: 600px;
```
See below.

File: curriculum/challenges/english/01-responsive-web-design/responsive-web-design-principles/make-an-image-responsive.english.md [hunk 3]
```diff
@@ -8,36 +8,32 @@ forumTopicId: 301140
...
 ```css
 img {
   max-width: 100%;
-  display: block;
   height: auto;
 }
 ```
 
-The <code>max-width</code> property of 100% scales the image to fit the width of its container, but the image won't stretch wider than its original width. Setting the <code>display</code> property to block changes the image from an inline element (its default), to a block element on its own line. The <code>height</code> property of auto keeps the original aspect ratio of the image.
+The `max-width` of `100%` will make sure the image is never wider than the container it is in, and the `height` of `auto` will make the image keep it's original aspect ratio.
```
Comment (ojeytonwilliams) [inline curriculum/challenges/english/01-responsive-web-design/responsive-web-design-principles/make-an-image-responsive.english.md]:
```suggestion
The `max-width` of `100%` will make sure the image is never wider than the container it is in, and the `height` of `auto` will make the image keep its original aspect ratio.
```

**Review by ojeytonwilliams — CHANGES_REQUESTED** [review]
One typo and some nitpicks.  The suggestions about width you can take or leave.

**Review by ojeytonwilliams — APPROVED** [review]

File: curriculum/challenges/english/01-responsive-web-design/responsive-web-design-principles/make-an-image-responsive.english.md [hunk 4]
```diff
@@ -8,36 +8,32 @@ forumTopicId: 301140
...
 ```css
 img {
   max-width: 100%;
-  display: block;
   height: auto;
 }
 ```
 
-The <code>max-width</code> property of 100% scales the image to fit the width of its container, but the image won't stretch wider than its original width. Setting the <code>display</code> property to block changes the image from an inline element (its default), to a block element on its own line. The <code>height</code> property of auto keeps the original aspect ratio of the image.
+The `max-width` of `100%` will make sure the image is never wider than the container it is in, and the `height` of `auto` will make the image keep its original aspect ratio.
```
Comment (thecodingaviator) [inline curriculum/challenges/english/01-responsive-web-design/responsive-web-design-principles/make-an-image-responsive.english.md]: How about this?
```suggestion
`max-width:100%;` will make sure the image is never wider than the container it is in, and `height: auto;` will make the image keep its original aspect ratio.
```

File: curriculum/challenges/english/01-responsive-web-design/responsive-web-design-principles/make-an-image-responsive.english.md [hunk 5]
```diff
@@ -68,13 +72,17 @@ tests:
...
+  height: auto;
+}
+
+img {
+  width: 600px;
+}
 </style>
 
+<img class="responsive-img" src="https://s3.amazonaws.com/freecodecamp/FCCStickerPack.jpg" alt="freeCodeCamp stickers set">
 <img src="https://s3.amazonaws.com/freecodecamp/FCCStickerPack.jpg" alt="freeCodeCamp stickers set">
```
Comment (thecodingaviator) [inline curriculum/challenges/english/01-responsive-web-design/responsive-web-design-principles/make-an-image-responsive.english.md]: Do we want to maybe remove this line from the solution? I haven't tried this so not sure if it'll work without this line.

File: curriculum/challenges/english/01-responsive-web-design/responsive-web-design-principles/make-an-image-responsive.english.md (hunk 4 above)
Comment (moT01) [reply curriculum/challenges/english/01-responsive-web-design/responsive-web-design-principles/make-an-image-responsive.english.md]: Either way would work here, but I prefer how it is now - seems more readable.

File: curriculum/challenges/english/01-responsive-web-design/responsive-web-design-principles/make-an-image-responsive.english.md (hunk 5 above)
Comment (moT01) [reply curriculum/challenges/english/01-responsive-web-design/responsive-web-design-principles/make-an-image-responsive.english.md]: Not sure why we would remove it. It's part of the seed code so it should stay in the solution, right?

**Comment by moT01** [comment]
Any second thoughts on these @thecodingaviator?

File: curriculum/challenges/english/01-responsive-web-design/responsive-web-design-principles/make-an-image-responsive.english.md (hunk 5 above)
Comment (thecodingaviator) [reply curriculum/challenges/english/01-responsive-web-design/responsive-web-design-principles/make-an-image-responsive.english.md]: Yeah. Sounds fine. I missed it somehow.

**Review by thecodingaviator — APPROVED** [review]
On second thoughts, LGTM!

**Comment by thecodingaviator** [comment]
Thank you for your contribution to the page! 👍
We're happy to accept these changes, and look forward to future contributions. 📝

File: curriculum/challenges/english/01-responsive-web-design/responsive-web-design-principles/make-an-image-responsive.english.md (hunk 5 above)
Comment (Paul-bee78) [reply curriculum/challenges/english/01-responsive-web-design/responsive-web-design-principles/make-an-image-responsive.english.md]: I am still unable to get this to work using a Microsoft Edge browser

Also, when working through the task it has no mention of using <code>display</code> nor any mention of applying the 'block' setting.

**Comment by RandellDawson** [comment]
This PR should not have been merged.   We are on a freeze of merging any English challenge in the Responsive Web Design certification.

**Comment by raisedadead** [comment]
> This PR should not have been merged. We are on a freeze of merging any English challenge in the Responsive Web Design certification to prevent the Chinese version from getting out of sync.

The code freeze came 10 days after this was merged: https://github.com/freeCodeCamp/freeCodeCamp/issues/38308#issue-573548954

> Please revert the changes.

I think we should create a fresh issue for the chinese team, if i18n needs updating instead.

**Comment by RandellDawson** [comment]
@raisedadead This is not an issue.  I misread the date of the merge.

**Pull request merged**